### PR TITLE
BZ2055275:adds an admonition

### PR DESCRIPTION
**Pull request by Amrita42**
- Applies to 4.8 and above
- [Bugzilla](https://bugzilla.redhat.com/show_bug.cgi?id=2055275)
- [Preview](https://53864--docspreview.netlify.app/openshift-enterprise/latest/openshift_images/samples-operator-alt-registry.html#installation-restricted-network-samples_samples-operator-alt-registry)
- @xiuwang ptal

### CODE REVIEW

**Comment by fbm3307** [comment]
@Amrita42 looks good to me

File: modules/installation-restricted-network-samples.adoc [hunk 1]
```diff
@@ -29,6 +29,11 @@ part of the install payload, are not managed by the Cluster Samples Operator. Th
 not addressed in this procedure.
 ====
 
+[IMPORTANT]
+====
+The Cluster Samples Operator must be set to `Managed` in a disconnected environment, to install the image streams you have mirrored.
```
Comment (skrthomas) [inline modules/installation-restricted-network-samples.adoc:34]:
```suggestion
The Cluster Samples Operator must be set to `Managed` in a disconnected environment. To install the image streams you have a mirrored registry.
```

**Review by skrthomas — COMMENTED** [review]
A couple of feedback items in one comment. Otherwise, LG!

**Comment by xiuwang** [comment]
LGTM

File: modules/installation-restricted-network-samples.adoc [hunk 2]
```diff
@@ -29,6 +29,11 @@ part of the install payload, are not managed by the Cluster Samples Operator. Th
 not addressed in this procedure.
 ====
 
+[IMPORTANT]
+====
+The Cluster Samples Operator must be set to `Managed` in a disconnected environment. To install the image streams, you have a mirrored registry.
```
Comment (gaurav-nelson) [inline modules/installation-restricted-network-samples.adoc:34]: **question:** @Amrita42 is this `mirrored` registry or mirror registry. 

Based on the links from Bugzilla I'm wondering if this is correct. I'll be happy to merge if this is the correct usage here.

Comment (Amrita42) [reply modules/installation-restricted-network-samples.adoc:34]: We have usage of mirrored registry for example >> https://docs.openshift.com/container-platform/4.11/installing/installing_bare_metal_ipi/ipi-install-prerequisites.html

Comment (Amrita42) [reply modules/installation-restricted-network-samples.adoc:34]: please go ahead with the merge. thanks

**Pull request merged**